index.html

```diff
@@ -14,12 +14,12 @@
 
 <body>
     <main>
-        <header>
+        <header class="flex align-center justify-center">
```

Comment:
I think here a `text-align: center` is enough. No need to make it flex just center some text.